apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/cache.ex

```diff
@@ -37,7 +37,7 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Cache do
           {key, _value} = Enum.random(map)
           Map.delete(map, key)
 
-        is_map(map) ->
+        true ->
           map
```

Comment:
This module was actually wrote to be quite general, I would rather add the `true` case and raise there:
```elixir
        is_map(map) -> map
        true -> raise "unexpected cache object #{inspect(map)}"
```